docs/source/commandline/commandline-all.rst

```diff
@@ -1931,33 +1931,63 @@ Core Input Options
 
 **-joystick_map** *<map>* / **-joymap** *<map>*
 
-    Controls how joystick values map to digital joystick controls. MAME accepts
-    all joystick input from the system as analog data.  For true analog
-    joysticks, this needs to be mapped down to the usual 4-way or 8-way digital
-    joystick values.  To do this, MAME divides the analog range into a 9x9 grid.
-    It then takes the joystick axis position (for X and Y axes only), maps it to
-    this grid, and then looks up a translation from a joystick map.  This
+    Controls how joystick values map to digital joystick controls.   
```

Comment:
This should explicitly say that it applies to *analog* joysticks – it doesn’t affect digital joytsticks.

Reply:
Yeah, missed that on the initial pass. Some of this comes from the original text that was in there. Fixed.